ale_linters/elixir/credo.vim

```diff
@@ -40,8 +40,15 @@ endfunction
 function! ale_linters#elixir#credo#GetCommand(buffer) abort
     let l:project_root = ale#handlers#elixir#FindMixProjectRoot(a:buffer)
 
-    return ale#path#CdString(l:project_root)
-    \ . ' mix help credo && mix credo suggest --format=flycheck --read-from-stdin %s'
+    if get(g:, 'ale_elixir_credo_strict', 0)
+        let l:credo_cmd = ' mix help credo && mix credo --strict --format=flycheck --read-from-stdin %s'
+
+        return ale#path#CdString(l:project_root) . l:credo_cmd
+    else
+        let l:credo_cmd = ' mix help credo && mix credo suggest --format=flycheck --read-from-stdin %s'
```

Comment:
Instead of replacing the whole string, replace just the one part of it we need to change based on the setting. Have a look at some of the other linter files.

Reply:
On it.  adding tests also.

Reply:
@w0rp Fixed.  Moved option generation to function to return correct value and concat that into the string command.  Also added `tests/command_callback/test_elixir_credo.vader` file.  I had a hell of a time getting it to work as you can see from the commits.  Please let me know if you have suggestions to make it better/correct, first Vader test I have ever written.